src/lua/bindings/event.cpp

```diff
@@ -1,5 +1,6 @@
 #pragma once
 #include "event.hpp"
+
```

Comment:
extra space?

Reply:
no, that's just how clang format does it, and it make sense as the first one is the mirroring header file

Reply:
Ah yes okay